data/rancher/ui_test.py

```diff
@@ -0,0 +1,44 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+import time
+
+driver = webdriver.Firefox()
```

Comment:
Have you considered enclosing this within `if __name__ == '__main__'` ?

    if __name__ == '__main__' :
        driver = webdriver.Firefox()
        ....

Reply:
Thanks done